src/csvcubed/utils/skos/codelist.py

```diff
@@ -63,6 +65,32 @@ def get_codelist_col_title_by_property_url(
     return results[0].column_title
 
 
+def get_codelist_unique_identifier_from_primary_key(
```

Comment:
Can you be clear it's actually the unique identifier's column title you're getting here? Or perhaps make this function more general so that it's clear that it's actually just taking a column's name and returning the column title?